docs/docs/cmd/teams/meeting/meeting-create.mdx

```diff
@@ -0,0 +1,241 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# teams meeting create
+
+Create a new online meeting
+
+## Usage
+
+```sh
+m365 teams meeting create [options]
```

Comment:
I think I overlooked this in the original specs: we try to stick to a specific set of verbs: add, remove, get, list, set.

So let's change this to `add` instead of `create`

Reply:
Ok I have also renamed all files to use add verb